syntax/leaninfo.vim

```diff
@@ -5,9 +5,9 @@ syn match leanInfoGoalVDash "^⊢"
 
 syn match leanInfoExpectedType "^▶ expected type.*"
 
-syn match leanInfoError "^▶.*: error:$"
-syn match leanInfoWarning "^▶.*: warning:$"
-syn match leanInfoInfo "^▶.*: information:$"
+syn match leanInfoError "^▶.*: Error:$"
```

Comment:
I have to admit I personally prefer lowercase titles (because they don't stand out that much visually), but I'm willing to compromise on that.

However, the titles should stay consistent.  If "Error" is now capitalized, then "Expected type"  (and "Goals accomplished") should be capitalized as well.

Reply:
Fair, agreed, reverted!